doc-content/enterprise-only/integration/sso-user-add-proc.adoc

```diff
@@ -2,7 +2,7 @@
 = Adding {PRODUCT} users
 Before you can use RH-SSO to authenticate {CENTRAL} or {KIE_SERVER}, you must add users to the realm that you created. To add new users and assign them a role to access {PRODUCT}, complete the following steps:
 
-. Log in to the RH-SSO Admin Console and open the realm to which you want to add a user.
+. Log in to the RH-SSO Admin Console and open the realm that you want to add a user to.
```

Comment:
Haha. I too defy the prescriptive antics against ending in a prep, but I've rarely seen someone proactively revert something to end in one. :) Fine by me.

Reply:
Haha, we were discussing prepositions during our BA workshop yesterday and we used this step as an example. I thought "why not adopt it". :)